<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLog</class>
 <widget class="QDialog" name="DialogLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOG</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLog">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEditLog">
           <property name="toolTip">
            <string>Application and debug log info shows here</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximumBlockCount">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxDisplayDebugInfo">
             <property name="toolTip">
              <string>Displays indepth debug info</string>
             </property>
             <property name="text">
              <string>Display Debug Info</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Max log size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMaxLogSize">
             <property name="toolTip">
              <string>Log is trimmed to most recent number of lines</string>
             </property>
             <property name="inputMask">
              <string>9999</string>
             </property>
             <property name="text">
              <string>5000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClearLog">
             <property name="toolTip">
              <string>Clears text</string>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Critical Notifications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditCriticalNotifications"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClearNotifications">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClearLog</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEditLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClearNotifications</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEditCriticalNotifications</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
